modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/UniqueNonEmptyContracts.swift

```diff
@@ -45,9 +45,9 @@ extension UniqueNonEmptyContracts {
             //NOTE: safe check to avoid incompatible contract matching with token type, blockout api returns erc20 and erc721 for same url, so we need to filter retults
             switch tokenType {
             case .erc20:
-                guard json["tokenID"].string == nil else { return nil }
+                guard json["tokenID"].stringValue.isEmpty && json["tokenValue"].stringValue.isEmpty else { return nil }
```

Comment:
What does `json["tokenValue"]` represent in the JSON for various token types?

Reply:
`tokenValue` supported only for erc1155, and represents transferred token amount
here is docs. https://docs.etherscan.io/api-endpoints/accounts#get-a-list-of-erc1155-token-transfer-events-by-address